eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/CircuitBreaker.scala

```diff
@@ -37,7 +38,7 @@ private class CircuitBreakerSettings(config: Config) {
  *
  * @see [[EventLogSPI.write]]
  */
-class CircuitBreaker(logProps: Props, batching: Boolean) extends Actor {
+class CircuitBreaker(logProps: Props, batching: Boolean, logId: String) extends Actor {
```

Comment:
Publishing `ServiceEvent`s directly (as mentioned in https://github.com/RBMHTechnology/eventuate/pull/293/commits/8bb8b80f602869fc76bf82ca409f5e2639c0b4b0#r75808663) would also prevent that the `logId` argument might differ from the logId val encoded in `logProps`.